wgsl/index.bs

```diff
@@ -12,7 +12,7 @@ Text Macro: UNSIGNEDINTEGRAL u32 or vec|N|&lt;u32&gt;
 Text Macro: SIGNEDINTEGRAL i32 or vec|N|&lt;i32&gt;
 Text Macro: INTEGRAL i32, u32, vec|N|&lt;i32&gt;, or vec|N|&lt;u32&gt;
 Text Macro: FLOATING f32 or vec|N|&lt;f32&gt;
-Ignored Vars: i, e, e1, e2, e3, eN, N, M, v, Stride, Offset, Align, Extent, S, T, T1
+Ignored Vars: i, e, e1, e2, e3, eN, s1, s2, sn, N, M, v, Stride, Offset, Align, Extent, B, S, T, T1
```

Comment:
I don't see any row where s1/s2/sn are defined and not used. Why do they have to be marked Ignored?

Reply:
I added cases for the ones that errored out.  I'll check again.

Reply:
It occurs because of the rowspans.  The second row of the rowspans for `loop` and `switch` only have one instance of |s1 and |s2| (for loop), and |s1| and |sn| (for switch)

```
bikeshed --die-on=everything spec index.bs
WARNING: The following <var>s were only used once in the document:
  's1', in algorithm 'loop with continuing with break behavior'
  's2', in algorithm 'loop with continuing with break behavior'
  's1', in algorithm 'switch with break behavior'
  'sn', in algorithm 'switch with break behavior'
If these are not typos, please add an ignore='' attribute to the <var>.
 ✘  Did not generate, due to fatal errors
```

Reply:
got it, thanks